--- node_modules/prosemirror-model/dist/index.js	2019-06-28 15:44:29.609149263 +0200
+++ node_modules/prosemirror-model/dist/index.js.new	2019-06-28 15:45:18.112303133 +0200
@@ -1491,7 +1491,7 @@
     var this$1 = this;
 
   for (var i = 0; i < this.next.length; i += 2)
-    { if (this$1.next[i] == type) { return this$1.next[i + 1] } }
+    { if (this$1.next[i].name == type.name) { return this$1.next[i + 1] } }
   return null
 };
 
