--- node_modules/prosemirror-view/dist/index.js.orig	1985-10-26 09:15:00.000000000 +0100
+++ node_modules/prosemirror-view/dist/index.js	2019-06-12 15:02:57.011725797 +0200
@@ -797,16 +797,16 @@
     { return {node: this.contentDOM, from: from, to: to, fromOffset: 0, toOffset: this.contentDOM.childNodes.length} }
 
   var fromOffset = -1, toOffset = -1;
-  for (var offset = 0, i = 0;; i++) {
+  for (var offset = base, i = 0;; i++) {
     var child = this$1.children[i], end = offset + child.size;
     if (fromOffset == -1 && from <= end) {
       var childBase = offset + child.border;
       // FIXME maybe descend mark views to parse a narrower range?
       if (from >= childBase && to <= end - child.border && child.node &&
           child.contentDOM && this$1.contentDOM.contains(child.contentDOM))
-        { return child.parseRange(from - childBase, to - childBase, base + childBase) }
+        { return child.parseRange(from, to, childBase) }
 
-      from = base + offset;
+      from = offset;
       for (var j = i; j > 0; j--) {
         var prev = this$1.children[j - 1];
         if (prev.size && prev.dom.parentNode == this$1.contentDOM && !prev.emptyChildAt(1)) {
@@ -818,7 +818,7 @@
       if (fromOffset == -1) { fromOffset = 0; }
     }
     if (fromOffset > -1 && to <= end) {
-      to = base + end;
+      to = end;
       for (var j$1 = i + 1; j$1 < this.children.length; j$1++) {
         var next = this$1.children[j$1];
         if (next.size && next.dom.parentNode == this$1.contentDOM && !next.emptyChildAt(-1)) {
