--- node_modules/prosemirror-view/dist/index.js.orig	2019-10-21 09:53:38.499595224 +0200
+++ node_modules/prosemirror-view/dist/index.js	2019-10-21 10:57:43.850916473 +0200
@@ -3058,7 +3058,7 @@
 var cssChecked = false;
 
 function checkCSS(view) {
-  if (cssChecked) { return }
+  if (cssChecked || view.cssChecked) { return }
   cssChecked = true;
   if (getComputedStyle(view.dom).whiteSpace == "normal")
     { console["warn"]("ProseMirror expects the CSS white-space property to be set, preferably to 'pre-wrap'. It is recommended to load style/prosemirror.css from the prosemirror-view package."); }
