--- node_modules/prosemirror-dropcursor/dist/dropcursor.js	2019-05-10 11:01:27.379852475 +0200
+++ node_modules/prosemirror-dropcursor/dist/dropcursor.js.new	2019-05-10 11:01:57.947454564 +0200
@@ -76,7 +76,7 @@
   var parent = this.editorView.dom.offsetParent;
   if (!this.element) {
     this.element = parent.appendChild(document.createElement("div"));
-    this.element.style.cssText = "position: absolute; z-index: 50; pointer-events: none; background-color: " + this.color;
+    this.element.style.cssText = "position: absolute; z-index: " + Number.MAX_SAFE_INTEGER + "; pointer-events: none; background-color: " + this.color;
   }
   var parentRect = parent == document.body && getComputedStyle(parent).position == "static"
       ? {left: -pageXOffset, top: -pageYOffset} : parent.getBoundingClientRect();
