--- node_modules/prosemirror-view/dist/index.js	2017-09-25 10:29:03.000000000 +0200
+++ node_modules.new/prosemirror-view/dist/index.js	2017-09-25 14:42:54.841108621 +0200
@@ -1001,7 +1001,7 @@
       var assign;
       ((assign = prosemirrorModel.DOMSerializer.renderSpec(document, node.type.spec.toDOM(node)), dom = assign.dom, contentDOM = assign.contentDOM));
     }
-    if (!contentDOM && !node.isText) { dom.contentEditable = false; }
+    if (!contentDOM && !node.isText && dom.nodeName != "IMG") { dom.contentEditable = false; }
 
     var nodeDOM = dom;
     dom = applyOuterDeco(dom, outerDeco, node);
