diff -Nur node_modules/prosemirror-view/dist/draw.js node_modules.new/prosemirror-view/dist/draw.js
--- node_modules/prosemirror-view/dist/draw.js	2016-10-04 08:43:12.000000000 +0200
+++ node_modules.new/prosemirror-view/dist/draw.js	2016-10-11 16:11:32.507132476 +0200
@@ -23,11 +23,7 @@
         dom.setAttribute("pm-size", node.nodeSize)
         if (node.isTextblock)
           { adjustTrailingHacks(serializer, dom, node) }
-        if (dom.contentEditable == "false") {
-          var wrap = document.createElement("div")
-          wrap.appendChild(dom)
-          dom = wrap
-        }
+        if (node.type && node.type.spec.readonly) dom.setAttribute("contenteditable", "false");
       }
 
       return dom
diff -Nur node_modules/prosemirror-view/dist/selection.js node_modules.new/prosemirror-view/dist/selection.js
--- node_modules/prosemirror-view/dist/selection.js	2016-10-04 08:43:12.000000000 +0200
+++ node_modules.new/prosemirror-view/dist/selection.js	2016-10-11 17:51:45.631291127 +0200
@@ -103,9 +103,8 @@
 
 function selectionToDOM(view, sel, takeFocus) {
   if (!view.hasFocus()) {
-    if (!takeFocus) { return }
     // See https://bugzilla.mozilla.org/show_bug.cgi?id=921444
-    else if (browser.gecko) { this.view.content.focus() }
+    if (takeFocus && browser.gecko) { this.view.content.focus() }
   }
 
   if (sel instanceof NodeSelection)
