--- node_modules/prosemirror-view/dist/index.js	2017-02-10 12:18:05.203431982 +0100
+++ node_modules/prosemirror-view/dist/index.js	2017-02-10 12:17:27.176199017 +0100
@@ -44,7 +44,7 @@
   this.content = document.createElement("div")
 
   if (place && place.appendChild) { place.appendChild(this.content) }
-  else if (place) { place(this.content) }
+  else if (place) { this.content = place(this.content) || this.content; }
 
   this.editable = getEditable(this)
   this.docView = docViewDesc(this.state.doc, computeDocDeco(this), viewDecorations(this), this.content, this)
